charactersheet/charactersheet/viewmodels/character/traits.js

```diff
@@ -54,6 +57,14 @@ function TraitsViewModel() {
         response(results);
     };
 
+    self.populateRace = function(label, value) {
+        self.blankTrait().race(value);
+    };
+
+    self.populateRaceEdit = function(label, value) {
```

Comment:
Our convention, so far, has been to not add pre-pop to the edit modal. What's the justification for this?